src/neuroglancer/shared_trackable_value.ts

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
This is very nice.  However, almost exactly the same code is now in src/neuroglancer/shared_watchable_value.ts, which was added as part of the prefetching support.